spec/workers/articles/bust_multiple_caches_worker_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+class CacheTestBuster
+  def self.bust(*); end
+end
```

Comment:
You don't need this anymore

Reply:
already removed 😂 . I did forget to push it